hack/make-rules/test-e2e-node.sh

```diff
@@ -71,14 +72,18 @@ if [[ ${parallelism} -gt 1 ]]; then
   ginkgoflags="${ginkgoflags} -nodes=${parallelism} "
 fi
 
-if [[ ${focus} != "" ]]; then
+if [[ ${focus} != "" && ${label_filter} == "" ]]; then
```

Comment:
Focus/skip/label-filter can be combined.

My suggestion only was that if a user is setting LABEL_FILTER, the script shouldn't second-guess their intention by applying defaults to SKIP.

A job which uses LABEL_FILTER then is in full control over test selection with a single expression.

Reply:
```suggestion
if [[ ${focus} != "" ]]; then
```

Reply:
I see, committed your suggestions for label_filter so it can be combined with skip/focus